common/shlibs

```diff
@@ -2841,7 +2841,7 @@ libyaz_icu.so.5 yaz-5.23.1_1
 libyaz.so.5 yaz-5.23.1_1
 libyaz_server.so.5 yaz-5.23.1_1
 libgcab-1.0.so.0 gcab-0.7_1
-libliveMedia.so.65 live555-2018.12.14_1
+libliveMedia.so.66 live555-2018.12.14_1
```

Comment:
According to `xrevshlib`, `vlc` depends on this library so it needs to be rebuilt on soname bump.

Reply:
So bump the revision on VLC too?